src/components/ToastNotification/ToastNotification.tsx

```diff
@@ -62,22 +64,37 @@ export const ToastNotification = ({
     className,
   );
 
+  // if both are defined, temporarily prefer the original (misspelled) prop for now, but throw a warning
```

Comment:
Why would we want or need to prefer the original (misspelled) prop when both are defined?